axelrod/strategies/axelrod_second.py

```diff
@@ -1508,3 +1508,42 @@ def strategy(self, opponent: Player) -> Action:
         return random_choice(self.prob_coop[(recent_history[-3],
                                              recent_history[-2],
                                              recent_history[-1])])
+
+
+class White(Player):
+    """
+    Strategy submitted to Axelrod's second tournament by Edward C Whtie (K72R)
+    and came in twelfth in that tournament.
```

Comment:
Looking at https://github.com/Axelrod-Python/axelrod-fortran/blob/master/src/axelrod_fortran/strategies.py#L214 it looks like it ranked 13th?

Reply:
It should say thirteenth, and actually the last few would be off.  I'll fix them.

Reply:
I got thrown off at Kluepfel, which had already been implemented since I've started doing these in order.

Reply:
> changes the behaviour of the fortran function

Changes how?  https://github.com/Axelrod-Python/TourExec/blob/v0.3.0/src/strategies/k72r.f#L12 must be rounding down the right-hand side, otherwise the 100% cooperation in the first ten moves (in the fingerprints) wouldn't make sense.

Reply:
If you pass a 0 (which is what we do when marking a function as `stochastic: False`) then the function seems to always defect. 

It's not an issue, just noting it. I think that essentially that `stochastic: ` key is whether or not the original Fortran code used the random number *anywhere*, even if as you've noticed here, it ended up not being relevant.